### PR TITLE
worked on Application

### PR DESCRIPTION
**Pull request by oscarsernarosero**

### CODE REVIEW

File: src/common/ErrorSelectors.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ An online Keccak-256 hash digester can be found at https://emn178.github.io/onli
 | 0x7a08a22d | CannotRemoveFunctionThatDoesNotExist(bytes4)                     |
 | 0x6fafeb08 | CannotRemoveImmutableFunction(bytes4)                            |
 | 0x192105d7 | InitializationFunctionReverted(address,bytes)                    |
-| 0x1da56a44 | OverMaxBalance()                                             |
+| 0x1da56a44 | OverMaxBalance()                                                 |
```
Comment (TJ-Everett) [inline src/common/ErrorSelectors.md:50]: Shane's OCD will love this catch

File: test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol [hunk 2]
```diff
@@ -17,7 +17,7 @@ contract RuleProcessorModuleFuzzTest is TestCommonFoundry, RuleCreation {
     /************************ PurchaseFeeByVolumeRule **********************/
 
     /// Simple setting and getting
-    function testPurchaseFeeByVolumeRuleSetting(uint8 addressIndex, uint256 volume, uint16 rate) public {
+    function testRuleProcessor_Diamond_PurchaseFeeByVolumeRuleSetting(uint8 addressIndex, uint256 volume, uint16 rate) public {
```
Comment (TJ-Everett) [inline test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol]: This and the next few test names have removed "fuzz" from them entirely but should still have it in there
i.e. testRuleProcessor_Diamond_fuzz_PurchaseFeeByVolumeRuleSetting

File: test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol [hunk 3]
```diff
@@ -43,7 +43,7 @@ contract RuleProcessorModuleFuzzTest is TestCommonFoundry, RuleCreation {
 
     /*********************** TokenMaxPriceVolatility ************************/
     /// Simple setting and getting
-    function testTokenMaxPriceVolatilitySetting(uint8 addressIndex, uint16 max, uint8 blocks, uint8 hFrozen) public {
+    function testRuleProcessor_Diamond_TokenMaxPriceVolatilitySetting(uint8 addressIndex, uint16 max, uint8 blocks, uint8 hFrozen) public {
```
Comment (TJ-Everett) [inline test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol]: Same here

File: test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol [hunk 4]
```diff
@@ -70,7 +70,7 @@ contract RuleProcessorModuleFuzzTest is TestCommonFoundry, RuleCreation {
 
     /*********************** TokenMaxTradingVolume Rule ************************/
     /// Simple setting and getting
-    function testTokenMaxTradingVolumeRuleSetting(uint8 addressIndex, uint16 max, uint8 hPeriod, uint64 _startTime) public {
+    function testRuleProcessor_Diamond_TokenMaxTradingVolumeRuleSetting(uint8 addressIndex, uint16 max, uint8 hPeriod, uint64 _startTime) public {
```
Comment (TJ-Everett) [inline test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol]: Same here

File: test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol [hunk 5]
```diff
@@ -99,7 +99,7 @@ contract RuleProcessorModuleFuzzTest is TestCommonFoundry, RuleCreation {
     /*********************** TokenMinTransactionSize ************************/
     /// Simple setting and getting
 
-    function testTokenMinTransactionSizeSetting(uint8 addressIndex, uint256 min) public {
+    function testRuleProcessor_Diamond_TokenMinTransactionSizeSetting(uint8 addressIndex, uint256 min) public {
```
Comment (TJ-Everett) [inline test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol]: And here

File: test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol [hunk 6]
```diff
@@ -198,7 +198,7 @@ contract RuleProcessorModuleFuzzTest is TestCommonFoundry, RuleCreation {
 
     /*********************** AccountApproveDenyOracle ************************/
     /// Simple setting and getting
-    function testAccountApproveDenyOracle(uint8 addressIndex, uint8 _type, address _oracleAddress) public {
+    function testRuleProcessor_Diamond_AccountApproveDenyOracle(uint8 addressIndex, uint8 _type, address _oracleAddress) public {
```
Comment (TJ-Everett) [inline test/protocol/diamond/fuzz/RuleProcessorModuleFuzz.t.sol]: and here

**Review by TJ-Everett — COMMENTED** [review]
Just found a few names that missed the fuzz key word. Holding approval for now while we decided the best path forward.

**Review by mpetersoCode55 — APPROVED** [review]
Looks great!

**Review by TJ-Everett — APPROVED** [review]
Approved

**Pull request merged**